### PR TITLE
First draft of EuroPython WASM summit report. TODO: @jezdez to add more notes.

### PR DESCRIPTION
**Pull request by ntoll**

### CODE REVIEW

**Comment by rajsite** [comment]
Are there to be recordings shared of those presentations? Those are a lot of interesting topics!

**Comment by ntoll** [comment]
Sadly no... and these were more like "seminars" than presentations... with lots of discussion and audience interactions - making videoing these things perhaps a challenge. There are photos tho!

File: meetings/2023/webassembly-summit-europython.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  Anaconda)
+
+These were presented in a "seminar fashion" where people were encouraged to
+respectfully interrupt, question, debate and otherwise constructively engage
+during the presentation of the subject matter.
+
+## Packaging debate
+
+The main focus for discussion in the afternoon was packaging in the context of
+web assembly. The TL;DR is something we've named the Lemburg Paradox (named
```
Comment (brettcannon) [inline meetings/2023/webassembly-summit-europython.md]:
```suggestion
WebAssembly. The TL;DR is something we've named the Lemburg Paradox (named
```

**Review by brettcannon — APPROVED** [review]

**Comment by ntoll** [comment]
I've added the group shot (nice for historical reasons).

Can someone please merge this for me, I don't have the required permissions to do so. Thank you!

**Comment by ntoll** [comment]
Also @jezdez do you want to add more from your notes..? (Perhaps good to pause until Janis has committed to this PR).

**Comment by brettcannon** [comment]
Sorry, read the "please merge" before I saw the "pause" comment since it was "below the fold" for me.

**Pull request merged**